### ISSUE TITLE
Buttons are sometimes blue when they should be gray

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-code-manager#1198]
### Describe the problem and steps to reproduce it:

* Load the `AccordionMenu` story from https://mozilla.github.io/addons-code-manager/?path=/story/accordionmenu--long-content

### What happened?

The buttons are blue.

### What did you expect to happen?

They should be gray.

### Anything else we should know?

The `AccordionMenu` on the main site still shows gray buttons so this is a very subtle specificity issue. It's possible that the order of DOM elements on the main site is such that the specificity works there but not in story book. In any case, this seems pretty fragile!

**Comment by ioanarusiczki** [comment]
@kumar303 
- is this reproducible only in the specified link ? https://mozilla.github.io/addons-code-manager/?path=/story/accordionmenu--long-content 

- on code manager dev or stage with Win10 and FF70 I've never seen the buttons being blue (browse or compare page). I also investigated a bit on a MacOS and reloaded the pages but the buttons stay gray.

**Comment by kumar303** [comment]
Yeah, the buttons were only blue in the story book but I think this is just because of a coincidence of DOM layout. In other words, it might have broken in the future without my change.

**Comment by ioanarusiczki** [comment]
@kumar303 Ok, it's grey for me in all places.
